packages/styles/src/carousel.scss

```diff
@@ -214,6 +213,7 @@ $fd-carousel-button-content-offset: 0.5rem !default;
 
     text-align: center;
     color: var(--sapPageFooter_TextColor);
+    font-size: var(--sapFontSmallSize);
```

Comment:
![Screenshot 2023-10-31 at 3 28 29 PM](https://github.com/SAP/fundamental-styles/assets/39598672/2f869500-36e0-4034-bf43-1b749a052c52)
![Screenshot 2023-10-31 at 3 28 40 PM](https://github.com/SAP/fundamental-styles/assets/39598672/353e44cd-9f37-4721-9393-d710acf8cfd3)

According to the specs we should have `--sapFontSize`

Reply:
I see in one design Figma that it's 12px so I will reach out the designers

Reply:
it should --sapFontSize (like it was)